examples/prepare-node-config/index.ts

```diff
@@ -69,15 +72,45 @@ async function main() {
   // get the core contracts from the transaction receipt
   const coreContracts = txReceipt.getCoreContracts();
 
+  const chainName = process.env.CHAIN_NAME ?? 'My Orbit Chain';
+  const parentChainRpc = process.env.PARENT_CHAIN_RPC ?? getRpcUrl(parentChain);
+
   // prepare the node config
   const nodeConfigParameters: PrepareNodeConfigParams = {
-    chainName: 'My Orbit Chain',
+    chainName,
     chainConfig,
     coreContracts,
     batchPosterPrivateKey: process.env.BATCH_POSTER_PRIVATE_KEY as `0x${string}`,
     validatorPrivateKey: process.env.VALIDATOR_PRIVATE_KEY as `0x${string}`,
     parentChainId: parentChain.id,
-    parentChainRpcUrl: getRpcUrl(parentChain),
+    parentChainRpcUrl: parentChainRpc,
+  };
+
+  const orbitSetupScriptConfigParams: OrbitSetupScriptConfigParams = {
+    networkFeeReceiver: chainConfig.arbitrum.InitialChainOwner,
+    infrastructureFeeCollector: chainConfig.arbitrum.InitialChainOwner,
+    staker: privateKeyToAccount(sanitizePrivateKey(process.env.VALIDATOR_PRIVATE_KEY!)).address,
+    batchPoster: privateKeyToAccount(sanitizePrivateKey(process.env.BATCH_POSTER_PRIVATE_KEY!)).address,
+    chainOwner: chainConfig.arbitrum.InitialChainOwner,
+    chainId: chainConfig.chainId,
+    chainName,
+    minL2BaseFee: 100000000, // todo: check default?
```

Comment:
Please review lines 97 and 100. Verify if the outputs for minL2BaseFee and utils are correct.